### ISSUE TITLE
debian package

### ISSUE THREAD

**Issue by PalinuroSec** [pooler/electrum-ltc#66]
hello, i am a developer of the parrot project (parrotsec.org) and i have recently packaged your electrum-ltc client

i have already integrated it in our debian derivative and i would like to maintain it in debian as well, and eventually provide you regularly updated dsc and deb files to be offered as an alternative download on your website for debian/ubuntu systems

here is where you can download the build results
https://archive.parrotsec.org/parrot/pool/main/e/electrum-ltc/

and here is where i am hosting the debianized source code
https://dev.parrotsec.org/parrot/electrum-ltc

of course mine was a 5 minutes job that can be improved with time, but i think it is a good starting point to further spread your software

**Comment by cavapoo2** [comment]
Hi Palin. I think this is great idea. How can it be put into the main official debian packages do you know the process ?. Quite a lot of people use tails os electrum wallet for bitcoin, to get electrum-ltc into Tails OS it needs to be available in debian.

**Comment by cavapoo2** [comment]
https://www.debian.org/devel/wnpp/#l2 . Probably needs @pooler to help with signing (gpg).

**Comment by PalinuroSec** [comment]
we need a debian maintainer or debian developer to embrace the package and maintain it in the archive, otherwise we can maintain it as free debian contributors, but we would need a DD to review and upload the package for us 

a good idea would be to involve the maintainers of the electrum bitcoin package and collaboratively maintain both the projects

**Comment by cavapoo2** [comment]
not sure which is easier, getting a debian dev to help out or involve the maintainers of electrum  / @pooler. I'm happy to help in anyway. Its sounds easier for them to just sign / verify. Can it be part of the github tree, so when they build it, some scripts auto gen the deb packages ( in a new folder this is all new to me).

**Comment by PalinuroSec** [comment]
yes, the d/ folder can be part of the upstream repo if we accept to be the uploaders of the debian package (i would love to maintain it, but i am just an external contributor)

**Comment by jidanni** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884299
